hummingbot/strategy/arbitrage/start.py

```diff
@@ -34,5 +35,5 @@ def start(self):
     self.market_trading_pair_tuples = [MarketTradingPairTuple(*primary_data), MarketTradingPairTuple(*secondary_data)]
     self.market_pair = ArbitrageMarketPair(*self.market_trading_pair_tuples)
     self.strategy = ArbitrageStrategy(market_pairs=[self.market_pair],
-                                      min_profitability=min_profitability,
+                                      min_profitability=Decimal(repr(min_profitability)),
```

Comment:
It would be more effective to add a "Decimal" type in `config_helpers.py` s.t. you don't need to do this imprecise type cast here.